spec.md

```diff
@@ -887,6 +887,14 @@ message VolumeCapability {
       // Can be published as read/write at multiple nodes
       // simultaneously.
       MULTI_NODE_MULTI_WRITER = 5;
+
+      // Can only be published once as read/write at a single workload
+      // on a single node, at any given time.
+      SINGLE_NODE_SINGLE_WRITER = 6 [(alpha_enum_value) = true];
```

Comment:
CO's using the experimental capability `SINGLE_NODE_MULTI_WRITER` should probably avoid specifying `SINGLE_NODE_WRITER`, yes? if for no other reason than for clarity. if so, please mention that here.

Reply:
Yes, COs should not use `SINGLE_NODE_WRITER` in this case. Added this to the description.